tools/wake-format/emitter.cpp

```diff
@@ -446,7 +415,28 @@ wcl::doc Emitter::walk_ideq(ctx_t ctx, CSTElement node) { return walk_placeholde
 
 wcl::doc Emitter::walk_if(ctx_t ctx, CSTElement node) { return walk_placeholder(ctx, node); }
 
-wcl::doc Emitter::walk_import(ctx_t ctx, CSTElement node) { return walk_placeholder(ctx, node); }
+wcl::doc Emitter::walk_import(ctx_t ctx, CSTElement node) {
+  assert(node.id() == CST_IMPORT);
+  return formatter()
+      .token(TOKEN_KW_FROM)
+      .ws()
+      .walk(CST_ID, WALK(walk_identifier))
+      .ws()
+      .token(TOKEN_KW_IMPORT)
+      .ws()
+      .fmt_if(CST_KIND, formatter().walk(CST_KIND, WALK(walk_kind)).ws())
+      .fmt_if(CST_ARITY, formatter().walk(CST_ARITY, WALK(walk_arity)).ws())
+      // clang-format off
+      .fmt_if_else(
+          TOKEN_P_HOLE,
+          formatter().walk(TOKEN_P_HOLE, WALK(walk_token)),
+          formatter().fmt_while(
+              CST_IDEQ,
+              formatter().walk(CST_IDEQ, WALK(walk_ideq)).fmt_if(TOKEN_WS, formatter().ws())))
```

Comment:
Can you define a local called something like "id_list_fmt" above this and pass it in? I think it will make the formatting better